app/src/main/java/com/github/se/eventradar/ui/hosting/Hosting.kt

```diff
@@ -70,7 +70,17 @@ fun HostingScreen(
   GetUserLocation(context, viewModel::onUserLocationChanged)
 
   ConstraintLayout(modifier = Modifier.fillMaxSize().testTag("hostingScreen")) {
-    val (logo, title, divider, searchfilter, filter, eventList, eventMap, bottomNav, buttons) =
+    val (
```

Comment:
not sure why it has been formatted like this in comparison to before hahah but not counted as new lines in SOnarCLoud so all good

Reply:
Not sure either, kfmt did it